omnibus/config/templates/datadog-agent/upstart_debian.sysprobe.conf.erb

```diff
@@ -22,7 +23,9 @@ pre-start script
 end script
 
 script
-  exec <%= install_dir %>/embedded/bin/system-probe --config=<%= etc_dir %>/system-probe.yaml --pid=<%= install_dir %>/run/system-probe.pid
+  if [ -f <%= etc_dir %>/system-probe.yaml ]; then
```

Comment:
I think this condition should be run in the pre-start script:
```
pre-start script
  test -f /etc/system-probe.yaml || { stop ; exit 0; }

  ...
end script
```
It's more idiomatic (see [the Upstart doc](http://upstart.ubuntu.com/cookbook/#stop-a-job-from-running-if-a-pre-start-condition-fails)) and we don't try to mount `/sys/kernel/debug` for nothing (less things we run as root that way).